packages/database/src/changeHandlers/ChangeHandler.js

```diff
@@ -47,10 +47,12 @@ export class ChangeHandler {
 
   /**
    * @abstract
+   * @protected
    */
-  handleChanges = async () => {
+  // eslint-disable-next-line no-unused-vars
+  async handleChanges(change) {
```

Comment:
I personally quite prefer to use this notation for instance methods of classes, though I'm not entirely sure of the differences...

Reply:
The key difference is that when passed in to another function as a callback, the arrow function will bind the correct class instance as `this`, which can be super important is specific situations and usually my motivation for going with it. It must be working without though, so I guess this isn't one of those situations!

Reply:
Ah damnit! The syntax I find uglier is the better performing one 😭 

Hmmm I have run into that issue once or twice before... Okay, I'll soften my stance on this. Thanks for mentioning it!